### ISSUE TITLE
Install Failure with Error Message Not Seen Here

### ISSUE THREAD

**Issue by TankMan649** [152334H/tortoise-tts-fast#72]
So I am posting the error message because it seems like I am not the only one who has had issues installing this.  However, what I haven't seen is my specific error message mentioned. This is what I am getting:

TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

Just as an FYI, I am running Debian 11 so the solutions for Windows did not really help but thanks as it did help in some ways.  Thanks.

**Comment by JoaCHIP** [comment]
No, you're not the only one who's having issues installing this.

Yesterday I spent over four hours trying to install this (and various possible dependencies) on Windows and a 3070 rtx card. I never got it to recognize Cuda. My biggest success was making it run on one single CPU core - which was of course rather slow.

My impression is that you need an intricate knowledge of how python development environments and python-specific package managers are working. Add to that a knowledge of Cuda development.

I wish there were either a much more fail-safe and complete install guide, or a precompiled binary installer for each OS. Like a good old "setup.exe" for Windows.
